source/extensions/omni.isaac.lab/omni/isaac/lab/sim/spawners/from_files/from_files.py

```diff
@@ -132,26 +131,31 @@ def spawn_ground_plane(
     Raises:
         ValueError: If the prim path already exists.
     """
+    # Get current stage
+    stage = omni.usd.get_context().get_stage()
     # Spawn Ground-plane
-    if not prim_utils.is_prim_path_valid(prim_path):
-        prim_utils.create_prim(prim_path, usd_path=cfg.usd_path, translation=translation, orientation=orientation)
+    if not stage.GetPrimAtPath(prim_path).IsValid():
```

Comment:
This check is duplicating logic that is in the newly created sim_utils.create_prim